src/registrar/templates/domain_base.html

```diff
@@ -0,0 +1,42 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block title %}Domain {{ domain.name }}{% endblock %}
+
+{% block content %}
+<div class="grid-container">
+  <div class="grid-row grid-gap">
+    <div class="grid-col-3">
+      {% include 'domain_sidebar.html' %}
+    </div>
+
+    <div class="grid-col-9">
+      <main id="main-content" class="grid-container">
+
+      {% if messages %}
+      {% for message in messages %}
+      <div class="usa-alert usa-alert--{{ message.tags }} usa-alert--slim margin-bottom-2">
```

Comment:
It continues to be a slight annoyance that blind users will not get notification of the styling. (A red warning message will "look" the same as a green success message when read by the screen reader.)

I think it's fine; we've just got to be careful when crafting our messages that the meaning and severity is completely conveyed by the text itself. CC @michelle-rago

Reply:
Unsure. We aren't making use of a messages file, so I think they are currently scattered around the codebase. Essentially, the errors are written wherever they are raised.

Use of a messages file to contain them would be a good idea, but I think our reasoning for not doing so was that we don't have any website translation planned.

Reply:
@michelle-rago These messages are produced in the code by `messages.success`, `messages.warning`, etc. in the appropriate flow in the code. See e.g. https://github.com/cisagov/getgov/blob/nmb/approved-users/src/registrar/views/domain.py#L75